waspc/src/Wasp/Generator/FileDraft.hs

```diff
@@ -40,6 +40,16 @@ instance Writeable FileDraft where
   write dstDir (FileDraftCopyDirFd draft) = write dstDir draft
   write dstDir (FileDraftTextFd draft) = write dstDir draft
 
+  getChecksum (FileDraftTemplateFd draft) = getChecksum draft
```

Comment:
oh some sweet TH can't wait to get sprinkled all over here :D

![cat_TH](https://user-images.githubusercontent.com/1536649/157630786-7f6fde2e-d8e5-4465-a8e8-30d4f25df56e.gif)

Reply:
Oh no :D!